Tribler/community/tunnel/main.py

```diff
@@ -11,6 +11,7 @@
 
 from twisted.internet.task import LoopingCall
 from twisted.internet.stdio import StandardIO
+from twisted.logger import globalLogPublisher
```

Comment:
@whirm `globalLogPublisher` is passed as root log object to the `clean_twisted_observers` method.